gdnative/src/class.rs

```diff
@@ -382,3 +384,181 @@ godot_class! {
         }
     }
 }
+
+// Class builder
+
+pub type GodotScriptMethodType = unsafe extern "C" fn(
```

Comment:
How about `GodotScriptMethodFn` (same thing for the other function pointer type aliases)?